docs/reference-guides/block-api/block-supports.md

```diff
@@ -329,16 +329,28 @@ supports: {
 -   Type: `Object`
 -   Default value: null
 -   Subproperties:
-    -   `padding`: type `boolean`, default value `false`
+    -   `padding`: type `boolean` or `array`, default value `false`
 
 This value signals that a block supports some of the CSS style properties related to spacing. When it does, the block editor will show UI controls for the user to set their values, if [the theme declares support](/docs/how-to-guides/themes/theme-support.md##cover-block-padding).
 
 ```js
 supports: {
-    padding: true, // Enable padding color UI control.
+    spacing: {
```

Comment:
Was `padding` shipped in WordPress 5.7?

- If so, we'll need to see how to support the old key as well.
- If not, this will still merit at least a breaking change note in the block-editor package (and bump its version accordingly).

Reply:
It’s quite possible I’ve gotten lost on this one. To me, when the API was stablized and then ready to be documented, it should have been `spacing.padding` within this doc file all along?

The padding block support flag was made stable in https://github.com/WordPress/gutenberg/pull/25788 changing from `__experimentalPadding` to `spacing.padding`. That PR was part of the Gutenberg v9.2 release and WordPress 5.7 included up to v9.9 of Gutenberg.

Happy to be pointed in the right direction if that is not correct as well as make any required changes.

Reply:
Thanks for sharing that. I've checked and this is fine, it's actually fixing a bug we had in the docs.

It looks like we used `spacing.padding` when it was made stable, but the docs were updated incorrectly https://github.com/WordPress/gutenberg/pull/26771 https://github.com/WordPress/gutenberg/pull/26859